code/modules/mob/dead/new_player/sprite_accessories.dm

```diff
@@ -655,6 +655,10 @@
 	name = "Ronin"
 	icon_state = "hair_ronin"
 
+/datum/sprite_accessory/hair/royal_curls
```

Comment:
needs modularization, but I'm happy to do that for you!

Reply:
Which folder would you suggest the hairstyle goes in?